operator/config/istio/virtual_service.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: networking.istio.io/v1beta1
+kind: VirtualService
+metadata:
+  name: kyma-events
+  namespace: operator-system
+spec:
+  gateways:
+    - default/kyma-gateway
+  hosts:
+    - '*'
+  http:
+    - match:
+        - uri:
+            regex: \/v.*\/kyma\/event
+      route:
+        - destination:
+            host: lifecycle-manager-kyma-event-service
+            port:
+              number: 8082
+    - match:
+        - uri:
+            regex: \/v.*\/manifest\/event
+      route:
+        - destination:
+            host: operator-manifest-event-service
```

Comment:
who deploy this event service? module-manager( manifest-operator)? then the `operator-` prefix should also change.